security-force-monitor/sfm-cms
# Redirect downloads, update message files to accommodate new text

## Overview

This PR temporarily redirects downloads to a message that they've been disabled pending performance improvements.

Connects #722 

## Testing Instructions

* Run the app as described in the README (you should not have to load in any data for this)
* Try navigating to http://localhost:8000/download and confirm you're redirected to http://localhost:8000/download-disabled
